File: uri 1012.java
import java.util.Scanner;
 

public class Main {
 
    public static void main(String[] args)  {
     
     Scanner input=new Scanner(System.in);
     double a,b,c,d,e,f,g,h;
     a = input.nextDouble();
     b = input.nextDouble();
     c = input.nextDouble();
     
     d = (0.5*a*c);
     e = (3.14159*c*c);
     f = ((0.5*(a+b))*c);
     g = (b*b);
     h  =(a*b);
     
     System.out.printf("TRIANGULO: %.3f\n",d);
     System.out.printf("CIRCULO: %.3f\n",e);
     System.out.printf("TRAPEZIO: %.3f\n",f);
     System.out.printf("QUADRADO: %.3f\n",g);
     System.out.printf("RETANGULO: %.3f\n",h);
     
       
    }
 
}

File: uri 1012.c
#include<stdio.h>
#define PI 3.14159
int main()
{
    float A,B,C;
    double tri,c,tra,s,r;
    scanf("%f%f%f",&A,&B,&C);
    tri = 0.5*A*C;
    c = PI*(C*C);
    tra = (((A+B)/2)*C);
    s = B*B;
    r = A*B;
    printf("TRIANGULO: %0.3lf\n",tri);
    printf("CIRCULO: %0.3lf\n",c);
    printf("TRAPEZIO: %0.3lf\n",tra);
    printf("QUADRADO: %0.3lf\n",s);
    printf("RETANGULO: %0.3lf\n",r);
    return 0;
}
